modules/adformBidAdapter.js

```diff
@@ -38,6 +38,11 @@ export const spec = {
 
     request.push('stid=' + validBidRequests[0].auctionId);
 
+    if (bidderRequest && bidderRequest.gdprConsent) {
+      request.push('gdpr=' + bidderRequest.gdprConsent.consentRequired);
```

Comment:
Hi @Pupis, as part of the recent updates we made in the GDPR branch to support the 1.1 CMP spec, we've renamed the `bidderRequest.gdprConsent.consentRequired` field to `bidderRequest.gdprConsent.gdprApplies`.  Can you change the reference to the new variable in this line of code?

Thanks.